### PR TITLE
refactor(Directive): drop moduleId

### PR DESCRIPTION
**Pull request by vicb**
moduleId is only used by components to resolve urls.
Directives have no templates and do not need moduleId.

### CODE REVIEW

File: modules/angular2/src/core/metadata/directives.ts [hunk 1]
```diff
@@ -865,6 +843,26 @@ export class ComponentMetadata extends DirectiveMetadata {
...
+   * ```
+   * @Directive({
+   *   selector: 'someDir',
+   *   moduleId: module.id
+   * })
+   * class SomeDir {
+   * }
+   *
+   * ```
+   */
```
Comment (gkalpak) [inline modules/angular2/src/core/metadata/directives.ts]: Shouldn't _directive_  be _component_ in this block-comment ?

Comment (vicb) [inline modules/angular2/src/core/metadata/directives.ts]: Yep, thanks for catching

**Comment by IgorMinar** [comment]
lgtm

**Comment by IgorMinar** [comment]
I don't think that @tbosch will get to review this, but it looks go to me, so let's merge it - if issues are identified after merge we'll deal with them.

**Pull request closed without merging**